### PR TITLE
Check licensing on travis

### PR DESCRIPTION
**Pull request by apahim**
Let's try to avoid some interactions on the pull requests by checking
if some license strings are present in .py files.

### CODE REVIEW

**Comment by apahim** [comment]
@ldoktor could you take a look here? THe second commit is only for test purpose. I will remove it as soon as we agree on the first commit.

File: .travis.yml [hunk 1]
```diff
@@ -20,3 +20,15 @@ script:
     - inspekt lint
     - inspekt indent
     - inspekt style
+    - |
+        err=false
+        for file in $(git diff --name-only $TRAVIS_COMMIT_RANGE -- *.py)
+        do
+            echo -n Checking license in file $file...
+            inspekt license --copyright "Copyright" --author "Author" $file || err=true
```
Comment (ldoktor) [inline .travis.yml]: Nice, there is only one problem I see, inspekt only supports GPLv2+ license for now (even though it claims GPLv2). And we have GPLv2_ONLY code getting to avocado and others like LGPL might follow.

Comment (apahim) [inline .travis.yml]: Yes, just noticed that as well. I will send a PR for inspekt, adding more license types.

**Comment by apahim** [comment]
@ldoktor @clebergnu, what do you guys think about this version?

**Comment by apahim** [comment]
@ldoktor @clebergnu, what do you guys think about this version?

**Comment by apahim** [comment]
Closing as it's easier to check licensing while reviewing the code than using travis.

**Pull request closed without merging**